deliver/lib/deliver/upload_metadata.rb

```diff
@@ -462,14 +451,18 @@ def retry_if_nil(message, tries: 5, wait_time: 10)
       end
     end
 
+    # Checking if the metadata to update includes App Info
+    def is_updating_app_info(options)
```

Comment:
This function could look more Ruby-y 😊 simply:

```ruby
def updating_app_info?(options)
  LOCALISED_APP_VALUES.keys.any? { |key| options[key] }
end
```

`updating_app_info?` is the Ruby naming convention for functions that return boolean values (instead of e.g. `is…(…)`) 😃

Reply:
Thanks! Clearly, I'm not a ruby-ist :-)

Reply:
I just know a thing or two! Always happy to share knowledge and help others ❤️